src/api/routes/v1/index.ts

```diff
@@ -37,4 +38,6 @@ api.use('/network/validator/:publicKey', handleValidator)
 api.use('/network/validators/:param', handleValidators)
 api.use('/network/validators', handleValidators)
 
+api.use('/network/networks', handleNetworks)
```

Comment:
lol

Reply:
lol that's what I thought initially. Open to any suggestion if any

Reply:
I don't have a better option either, it's just funny

Reply:
why cant it be just `/networks`

Reply:
All the VHS endpoints are at `/v1/network/...`, that'd be a major change